code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -1,55 +1,101 @@
+						"<span class='suicide'>[user] is falling on the [src.name]! It looks like \he's trying to commit suicide.</span>"))
 	return (BRUTELOSS|FIRELOSS)
+	
+/obj/item/weapon/melee/energy/attack_self(mob/living/carbon/user)
+	if(user.disabilities & CLUMSY && prob(50))
+		user << "<span class='warning'>You accidentally cut yourself with [src], like a doofus!</span>"
+		user.take_organ_damage(5,5)
+	active = !active
+	if (active)
+		force = force_on
+		throwforce = throwforce_on
+		hitsound = 'sound/weapons/blade1.ogg'
+		if(attack_verb_on.len)
+			attack_verb = attack_verb_on
+		if(!_color)
+			icon_state = icon_state_on
+		else
+			icon_state = "sword[_color]"
+		w_class = w_class_on
+		playsound(user, 'sound/weapons/saberon.ogg', 35, 1) //changed it from 50% volume to 35% because deafness
+		user << "<span class='notice'>[src] is now active.</span>"
+	else
+		force = initial(force)
+		throwforce = initial(throwforce)
+		hitsound = initial(hitsound)
+		if(attack_verb_on.len)
+			attack_verb = list()
+		icon_state = initial(icon_state)
+		w_class = initial(w_class)
+		playsound(user, 'sound/weapons/saberoff.ogg', 35, 1)  //changed it from 50% volume to 35% because deafness
+		user << "<span class='notice'>[src] can now be concealed.</span>"
+	add_fingerprint(user)
+	return
 
 /obj/item/weapon/melee/energy/axe
 	name = "energy axe"
 	desc = "An energised battle axe."
 	icon_state = "axe0"
-	force = 40.0
-	throwforce = 25.0
-	throw_speed = 1
+	force = 40
+	force_on = 150
+	throwforce = 25
+	throwforce_on = 30
+	throw_speed = 3
 	throw_range = 5
-	w_class = 3.0
+	w_class = 3
+	w_class_on = 5
 	hitsound = "swing_hit"
 	flags = CONDUCT | NOSHIELD
 	origin_tech = "combat=3"
 	attack_verb = list("attacked", "chopped", "cleaved", "torn", "cut")
+	attack_verb_on = list()
 	sharp = 1
 	edge = 1
 
-
 /obj/item/weapon/melee/energy/axe/suicide_act(mob/user)
-		viewers(user) << "<span class='suicide'>[user] swings the [src.name] towards /his head! It looks like \he's trying to commit suicide.</span>"
-		return (BRUTELOSS|FIRELOSS)
+	user.visible_message("<span class='suicide'>[user] swings the [src.name] towards /his head! It looks like \he's trying to commit suicide.</span>")
+	return (BRUTELOSS|FIRELOSS)
 
 /obj/item/weapon/melee/energy/sword
-	var/hacked = 0
-	var/blade_color
-	color
 	name = "energy sword"
 	desc = "May the force be within you."
 	icon_state = "sword0"
-	force = 3.0
-	throwforce = 5.0
-	throw_speed = 1
+	force = 3
+	throwforce = 5
+	throw_speed = 3
 	throw_range = 5
-	w_class = 2.0
 	hitsound = "swing_hit"
 	flags = NOSHIELD
 	origin_tech = "magnets=3;syndicate=4"
-	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 	sharp = 1
 	edge = 1
+	var/hacked = 0
+	var/blade_color
+	
+/obj/item/weapon/melee/energy/sword/New()
+	if(_color == null)
+		_color = pick("red", "blue", "green", "purple")
+
+/obj/item/weapon/melee/energy/sword/IsShield()

```

Comment:
errr if I recall correctly this proc has much more affect here than it does on -tg-